app/controllers/observation_quality_experiments_controller.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class ObservationAccuracyExperimentsController < ApplicationController
+  layout "bootstrap"
+
+  def get_more_validators
+    experiment_id = params[:id]
+    @experiment = ObservationAccuracyExperiment.find( experiment_id )
+    @more_validators = @experiment.get_validator_names( limit: nil, offset: 20 )
+    respond_to do | format |
+      format.html { render partial: "additional_validators", locals: { validators: @more_validators } }
+    end
+  end
+
+  def show
+    @experiment = ObservationAccuracyExperiment.find( params[:id] )
+    @explorable = ( @experiment.validator_deadline_date < Time.now ) ||
+      ( logged_in? && ( current_user.is_admin? || is_curator_or_site_admin ) )
+
+    @validators = @experiment.get_validator_names( limit: 20, offset: 0 )
+
+    @tab = params[:tab] || "research_grade_results"
+    if @tab == "methods"
+      @candidate_validators, @mean_validator_count, @mean_sample_count = get_assignment_methods
+      @mean_validators_per_sample, @validators_per_sample, @validators_per_sample_ylim = get_validation_methods
+    else
+      @stats, @data, @precision_data = get_results_data
+    end
+    render "show"
+  end
+
+  private
```

Comment:
these private methods in this controller seem to be about gathering data from experiments. These would preferably be implemented as methods in the experiment model to get this logic out of the controller. The instance variables the methods depend on could be changed to method parameters